### PR TITLE
Upgrade to engine_cart 1.0

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by jcoyne** [comment]
@cam156, @awead, @hackmastera, @elrayle  it looks like a dependency is not pinned correctly, but I don't know which it might be. Since you have a running copy of sufia 6, can you report on your Gemfile.lock. Specifically looking for versions of blacklight, blacklight-access-controls, and hydra-head.

**Comment by awead** [comment]
@jcoyne from the Gemfile.lock of our production Sufia 6 instance:
``` bash
$ egrep "sufia|blacklight|hydra-head" Gemfile.lock 
  branch: sufia
      hydra-head (~> 9.1)
  remote: git://github.com/projecthydra/sufia.git
    sufia (6.7.0)
      blacklight (~> 5.16)
      blacklight-gallery (~> 0.1)
      blacklight_advanced_search (>= 5.1.4, < 6.0)
      sufia-models (= 6.7.0)
    sufia-models (6.7.0)
      hydra-head (< 9.6)
    blacklight (5.18.0)
    blacklight-gallery (0.4.1)
      blacklight (>= 5.12, < 7)
    blacklight_advanced_search (5.2.1)
      blacklight (~> 5.15)
      blacklight (~> 5.16)
      blacklight
      blacklight (~> 5.10)
      hydra-head (~> 9.1)
    hydra-head (9.5.0)
  blacklight (= 5.18.0)
  hydra-head (~> 9.5.0)
  sufia!
```

**Comment by hackmastera** [comment]
mine looks much like awead's: 
```
$ egrep "sufia|blacklight|hydra-head" Gemfile.lock 
  branch: sufia
  remote: git://github.com/projecthydra/sufia.git
    sufia (6.7.0)
      blacklight (~> 5.16)
      blacklight-gallery (~> 0.1)
      blacklight_advanced_search (>= 5.1.4, < 6.0)
      sufia-models (= 6.7.0)
    sufia-models (6.7.0)
      hydra-head (< 9.6)
    blacklight (5.16.4)
    blacklight-gallery (0.4.0)
      blacklight (~> 5.12)
    blacklight_advanced_search (5.2.1)
      blacklight (~> 5.15)
      blacklight (~> 5.10)
      blacklight
      blacklight (~> 5.10)
      hydra-head (~> 9.1)
    hydra-head (9.4.1)
      blacklight
  blacklight (= 5.16.4)
  blacklight-gallery (= 0.4.0)
  sufia!
```

File: sufia.gemspec [hunk 1]
```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'sass-rails'
 
   gem.add_dependency 'hydra-batch-edit', '~> 1.1'
+  gem.add_dependency 'hydra-head', '~> 9.5.0'
```
Comment (cbeer) [inline sufia.gemspec]: Ugh. We really need to be this pessimistic about this?

Comment (awead) [inline sufia.gemspec]: Yes, unfortunately, because of https://github.com/projecthydra/sufia/issues/1530

**Pull request merged**